python/paddle/fluid/reader.py

```diff
@@ -894,7 +895,7 @@ def _reader_thread_loop_for_singleprocess(self):
                 array = core.LoDTensorArray()
                 for item in sample:
                     if not isinstance(item, core.LoDTensor):
-                        self._check_input_array(item)
+                        item = self._check_input_array(item)
                         tmp = core.LoDTensor()
                         tmp.set(item, core.CPUPlace())
```

Comment:
Is there any efficiency difference between setting list or ndarray to a LoDTensor ?